cardano-sync/src/Cardano/Sync/LedgerState.hs

```diff
@@ -86,6 +88,7 @@ import           Ouroboros.Network.Block (HeaderHash, Point (..))
 import qualified Ouroboros.Network.Point as Point
 
 import qualified Shelley.Spec.Ledger.BaseTypes as Shelley
+import           Shelley.Spec.Ledger.Keys
```

Comment:
All imports from outside this repo should be qualified or explicit.

This makes it easier to figure out where a symbol comes from